SparkPost/2web2ui
# Upgrade Handlebars for Jest to fix security vulnerability

_There is no ticket.  I am on support this week and saw that this should be done._

### What Changed

The following is part of the audit report -- I removed all but "High" priority security vulnerabilities.  This PR will only upgrades handlebars to fix the one high priority vulnerability.

```
$ npm audit

                         === npm audit security report ===

  # Run  npm update handlebars --depth 5  to resolve 1 vulnerability
  ┌───────────────┬──────────────────────────────────────────────────────────────┐
  │ High          │ Prototype Pollution                                          │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ Package       │ handlebars                                                   │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ Dependency of │ jest [dev]                                                   │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ Path          │ jest > jest-cli > istanbul-api > istanbul-reports >          │
  │               │ handlebars                                                   │
  ├───────────────┼──────────────────────────────────────────────────────────────┤
  │ More info     │ https://npmjs.com/advisories/755                             │
  └───────────────┴──────────────────────────────────────────────────────────────┘

  found 4 vulnerabilities (1 low, 2 moderate, 1 high) in 112504 scanned packages
    run `npm audit fix` to fix 4 of them.
```

### How To Test

There is nothing to test, just confirm our test coverage reports are still generated when `npm run test-ci` is run locally and in our CI builds.  The reason there is nothing to test is because
[handlebars is only used by istanbul-reports for html reports](https://github.com/istanbuljs/istanbuljs/search?q=handlebars&unscoped_q=handlebars), istanbul-reports is only used by jest for generating test coverage reports.
